PVB Linx Groep 6/Assets/Code/Scripts/UniversalHealth.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum ObjectType
```

Comment:
Deze enum zou overbodig moeten zijn, we willen dat het health script universeel werkt zonder dat je vooraf moet aangeven op wat voor gameobject het zit